definitions/infra-host/golden_metrics.yml

```diff
@@ -12,6 +12,12 @@ cpuUsage:
       from: SystemSample
       eventId: entityGuid
       eventName: entityName
+    opentelemetry:
```

Comment:
This requires the telemetry to bring an `instrumentation.provider=opentelemtry` attribute. Is that actually happening?

Last time I talked with Julien I understood otel didn't bring this

Reply:
yes. that one. nice :D